src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PERSON_TABLE_COLUMNS } from '../../constants/constants';
+import { PersonItem } from '../PersonItem/PersonItem';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = props => {
+  const { people } = props;
+
+  const { slug: currentSlug } = useParams();
+
+  const findParent = (parentName: string | null): Person | null => {
+    return people.find(person => person.name === parentName) || null;
+  };
+
+  const preparedPeople = people.map(person => {
+    return {
+      ...person,
+      mother: findParent(person.motherName),
+      father: findParent(person.fatherName),
+    };
+  });
```

Comment:
Prepare people array in the parent component, the `PeopleTable` is a presentational component and shouldn't have any logic